backend/jest.config.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
     '@Repositories': '<rootDir>/src/repositories',
     '@Middlewares': '<rootDir>/src/middlewares',
     '@Config': '<rootDir>/src/config.ts',
+    '@Logger': '<rootDir>/src/logging.ts',
```

Comment:
Perhaps putting this in a separte folder is better? Just to conform to other folders. Up to you on this one tbh

Reply:
Personally, I think it's fine since it's just one file. @godwinpang  what do you think?

Reply:
yeah I think it's actually better as its own file since there really shouldn't be too much for logging, if our logging setup gets complicated enough then we can move it into its folder and still be protected by the @Logger abstraction